wp-tevko-responsive-images.php

```diff
@@ -301,31 +304,30 @@ function tevkori_get_src_sizes( $id, $size = 'thumbnail' ) {
 }
 
 /**
- * Filter for extending image tag to include srcset attribute.
+ * Filter for the_content to add sizes and srcset attributes to images.
+ *
+ * @since 3.0
  *
- * @see images_send_to_editor
- * @return string HTML for image.
+ * @param string $content The raw post content to be filtered.
  */
-function tevkori_extend_image_tag( $html, $id, $caption, $title, $align, $url, $size, $alt ) {
-	add_filter( 'editor_max_image_size', 'tevkori_editor_image_size' );
-
-	// Get the srcset attribute.
-	$srcset = tevkori_get_srcset_string( $id, $size );
-
-	remove_filter( 'editor_max_image_size', 'tevkori_editor_image_size' );
+function tevkori_filter_content_images( $content ) {
+	preg_match_all( '/src="([^"]*)"/', $content, $matches );
```

Comment:
@peterwilsoncc 

I am not a regex expert either, but I don't think they should be used to check if something is *not* there. If it's possible to write a regex like that it will probably not perform very well.